functionary/ui/templates/builder/build_detail.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+{% block content %}
+    <div class="column is-full">
+        <nav class="breadcrumb has-arrow-separator" aria-label="breadcrumbs">
+            <ul>
+                <li>
+                    <a href="{% url 'ui:build-list' %}">Build List</a>
+                </li>
+                <li class="is-active">
+                    <a href="#">{{ object.id }}</a>
+                </li>
+            </ul>
+        </nav>
+    </div>
+    <div class="column">
+        <div class="block">
+            <h1 class="title is-1">
+                <span class="icon"><i class="fa fa-wrench"></i></span>&nbsp;&nbsp;
+                {{ object.id }}
+            </h1>
+        </div>
+        <div class="column has-addons">
+            {% if object.created_at %}
+                <div class="field">
+                    <label class="label" for="desc">Created At:</label>
+                    <span class="ml-4">{{ object.created_at }}</span>
+                </div>
+            {% endif %}
+            {% if object.updated_at %}
+                <div class="field">
+                    <label class="label" for="desc">Updated At:</label>
+                    <span class="ml-4">{{ object.updated_at }}</span>
+                </div>
+            {% endif %}
+            {% if object.creator %}
+                <div class="field">
+                    <label class="label" for="desc">Creator:</label>
+                    <span class="ml-4">{{ object.creator }}</span>
+                </div>
+            {% endif %}
+            {% if object.status %}
+                <div class="field">
+                    <label class="label" for="desc">Status:</label>
+                    <span class="ml-4">{{ object.status }}</span>
+                </div>
+            {% endif %}
+            {% if object.environment %}
+                <div class="field">
+                    <label class="label" for="desc">Environment:</label>
```

Comment:
We don't need to display the environment here, since you are only ever looking at the builds for a single environment.  Basically you select an environment up front and then all data you are looking at is for that environment, so displaying the environment again anywhere ends up being redundant.  This redundancy shows up in other places currently and needs to be cleaned up, but we may as well clean it up here now.